Autopilot/AttitudeManager/IMU.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file IMU.h
+ * @author Ian Frosst
+ * @date March 16, 2017
+ * @copyright Waterloo Aerial Robotics Group 2017 \n
+ *   https://raw.githubusercontent.com/UWARG/PICpilot/master/LICENCE
+ */
+
+#ifndef IMU_H
+#define	IMU_H
+
+#define VN100 0
+#define MPU9250 1
+#define MPU6050 2
+
+#define USE_IMU VN100
+
+typedef struct IMU_data {
+    float roll, pitch, heading;
+    float rollRate, pitchRate, yawRate;
+} IMU_data;
+
+
+void initIMU();
```

Comment:
Adds docs to the functions here. Also in header files prototypes need to be declared with the void parameter, since in c just brackets means it takes any amount of paramaters. When you implement these functions in a c file you'll get double declaration errors. So make these accept void and create an empty IMU.c file that have the implementation of these functions